### ISSUE TITLE
Support for multiline comments

### ISSUE THREAD

**Issue by Mingun** [kevinmehall/rust-peg#235]
Currently if you try to write comment, that occupies multiply lines, that code won't compiled:
```rust
peg::parser! {
  /// Grammar
  /// second line of grammar documentation
  grammar G() for str {
    /// Rule
    /// second line of rule documentation
    rule R() = [_];
  }
}
```
Instead you got something like that:
```
error: expected one of "crate", "grammar", "pub"
 --> src\expressions.rs:7:3
  |
7 |   /// second line of grammar documentation
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
```
error: expected "cache"
  --> src\expressions.rs:10:5
   |
10 |     /// second line of rule documentation
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by kevinmehall** [comment]
Fixed in https://github.com/kevinmehall/rust-peg/pull/232, but I need to publish a release.

**Comment by Mingun** [comment]
Thanks! (Strange, why GitHub didn't suggest similar issue when I creating this....)

**Comment by kevinmehall** [comment]
Fix released in 0.6.3
